io.freetubeapp.FreeTube.metainfo.xml

```diff
@@ -3,15 +3,411 @@
   <id type="desktop-application">io.freetubeapp.FreeTube</id>
   <metadata_license>CC0-1.0</metadata_license>
   <name>FreeTube</name>
-  <summary>An Open Source YouTube app for privacy</summary>
+  <summary>An open-source YouTube app for privacy</summary>
+  <summary xml:lang="ar">تطبيق يوتيوب مفتوح المصدر للخصوصية</summary>
+  <summary xml:lang="az">Gizlilik üçün açıq mənbəli YouTube proqramı</summary>
+  <summary xml:lang="be">Праграма YouTube з адкрытым зыходным кодам для забеспячэння прыватнасці</summary>
+  <summary xml:lang="bg">Приложение за YouTube с отворен код за поверителност</summary>
+  <summary xml:lang="bn">গোপনীয়তার জন্য একটি ওপেন-সোর্স YouTube অ্যাপ</summary>
+  <summary xml:lang="ca">Una aplicació de YouTube de codi obert per a la privadesa</summary>
+  <summary xml:lang="cs">Open-source aplikace YouTube pro ochranu soukromí</summary>
+  <summary xml:lang="da">En open source YouTube-app til beskyttelse af personlige oplysninger</summary>
+  <summary xml:lang="de">Eine Open-Source-YouTube-App für den Datenschutz</summary>
+  <summary xml:lang="el">Μια εφαρμογή YouTube ανοιχτού κώδικα για προστασία απορρήτου</summary>
+  <summary xml:lang="en_AU">An open-source YouTube app for privacy</summary>
+  <summary xml:lang="en_GB">An open-source YouTube app for privacy</summary>
+  <summary xml:lang="en">An open-source YouTube app for privacy</summary>
```

Comment:
Please follow the guidelines, https://docs.flathub.org/docs/for-app-authors/metainfo-guidelines#metainfo-translations. This can be removed as it is the same as https://github.com/flathub/io.freetubeapp.FreeTube/pull/124/files#diff-1882824c517d6453f283660b986de519c7579fc2ce30621dd264d21bea9271e8R6

![firefox_lPtpXDOQnV](https://github.com/user-attachments/assets/e1251ed0-248b-410a-9cf6-91d4dd0ff366)

Reply:
Done